.ci/copy-site-to-sourceforge.sh

```diff
@@ -17,48 +17,44 @@ if [[ -z $PREV_RELEASE ]]; then
   exit 1
 fi
 
-SF_USER=romanivanov
-#############################
-echo "Please provide password for $SF_USER,checkstyle@shell.sourceforge.net"
-echo "exit" | ssh -t $SF_USER,checkstyle@shell.sourceforge.net create
-
-#####
+echo "Creating shell for $SF_USER@shell.sourceforge.net"
+ssh -i ~/.ssh/private_sourceforge_key -t "$SF_USER"@shell.sourceforge.net create
 
 mkdir -p .ci-temp
 cd .ci-temp
 rm -fr checkstyle.github.io
-echo "Clone by ssh only to avoid passwords on push"
-git clone git@github.com:checkstyle/checkstyle.github.io.git
-echo "clean up git files"
+echo "Cloning checkstyle.github.io repo"
+git clone https://github.com/checkstyle/checkstyle.github.io.git
+echo "Cleaning up git files"
 rm -rf checkstyle.github.io/.git
 rm -rf checkstyle.github.io/CNAME
-echo "Archiving ..."
+echo "Archiving"
 tar cfz checkstyle.github.io.tar.gz checkstyle.github.io
-echo "Uploading to sourceforge ..."
-scp checkstyle.github.io.tar.gz \
-  $SF_USER,checkstyle@shell.sourceforge.net:/home/project-web/checkstyle/
+echo "Uploading to sourceforge"
+scp -i ~/.ssh/private_sourceforge_key checkstyle.github.io.tar.gz \
+  "$SF_USER"@shell.sourceforge.net:/home/project-web/stoyank-checkstyle
```

Comment:
I will undo this when approved before merging.

Reply:
As you confirm testing, I will merge it.
I will test all action in release this Sunday.
thanks a lot for your work.

Reply:
Final test before reverting back from `stoyank-checkstyle`: https://github.com/stoyanK7/checkstyle/actions/runs/3541990560/jobs/5946940410
Website: https://stoyank-checkstyle.sourceforge.io/

Reply:
Done. Should be good to merge once CI passes.

Reply:
Do not forget to add the `SSH_KEY` secret variable with the private ssh key for Sourceforge.